BrickKit/bricks/src/main/java/com/wayfair/brickkit/brick/DataModel.kt

```diff
@@ -18,7 +18,7 @@ abstract class DataModel : Serializable {
      *
      * @param updateListener the listener to add
      */
-    fun addUpdateListener(updateListener: DataModelUpdateListener) {
+    open fun addUpdateListener(updateListener: DataModelUpdateListener) {
```

Comment:
Had to make these open so that they can be mocked in the main codebase (adding OpenForTesting made all subclasses also open (which broken private setters in those subclasses)